source/_integrations/zwave_js.markdown

```diff
@@ -45,5 +46,5 @@ available.
 
 As this integration is still in the early stages there are some important limitations to be aware of.
 
-- Platform support is currently limited to basic `binary_sensor`, `climate`, `cover`, `light`, `lock`, `sensor`, and `switch` entities.
+- Platform support is currently limited to basic `binary_sensor`, `climate`, `cover`, `fan`, `light`, `lock`, `sensor`, and `switch` entities.
```

Comment:
That's all the platforms so we can remove this line all together.

Reply:
```suggestion
```